go.mod

```diff
@@ -156,7 +156,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opensearch-project/opensearch-go/v2 v2.3.0
 	github.com/parquet-go/parquet-go v0.23.0
-	github.com/patrickmn/go-cache v0.0.0-20180815053127-5633e0862627
+	github.com/patrickmn/go-cache v2.1.0+incompatible
```

Comment:
It's rather unintuitive, but v2.1.0 is actually before the commit we are pulling.

See https://github.com/patrickmn/go-cache/compare/v2.1.0...5633e0862627.